api/sys_capabilities.go

```diff
@@ -12,7 +12,21 @@ func (c *Sys) CapabilitiesSelf(path string) ([]string, error) {
 	return c.Capabilities(c.c.Token(), path)
```

Comment:
This one isn't calling the WithContext version of itself like the others, but that's fine because it calls Capabilities which creates a context.Background(), right? Meaning CapabilitiesSelfWithContext is only used by those users who invoke it directly.

Reply:
Great catch! 👍 This is technically correct but it's good to standardize the code, I'll fix it.